README.md

```diff
@@ -108,6 +108,32 @@ For pagination, like filtering, we don't enforce any view-layer specifics. You s
 </nav>
 ```
 
+## Configuration
+
+### ActiveSet.configuration.on_asc_sort_nils_come
+
+Example usage in an initializer:
+
+```
+  ActiveSet.configure do |c|
+    c.on_asc_sort_nils_come = first
```

Comment:
missing the colon for the symbol here

Reply:
Thanks. Fixed in c0660303e00130282748da56c3300158752de35d